cypress/plugins/index.js

```diff
@@ -10,17 +10,19 @@
 
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
+const percyHealthCheck = require("../../dist/task");
 
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
 
   // Make it possible to log things to stdout by calling 'cy.task('log', 'some message to log').
   // Useful for development and debugging.
-  on('task', {
-    log (message) {
-      console.log(message)
-      return null
+  on("task", {
+    log(message) {
+      console.log(message);
+      return null;
     }
-  })
-}
+  });
+  on("task", percyHealthCheck);
```

Comment:
At first glance this looks like you're passing a function, which is not supported. Might be a little more clear to set this with an explicit object rather than rely on the module structure of require.

I'd vote for exporting the health check function as a default and changing this to be `{ percyHealthCheck }`, or including it in the other `on('task', ...)` above if possible.

Reply:
Upon looking over cypress docs on tasks more, it seems in their example of task registration we should be fine to rely on the module structure of require if we need the users to register the health check themselves. That way they can `require('@percy/cypress/task')` and pass that directly to `on('task', percyTasks)`

Reply:
Going to resolve this convo since we decided to move this out into the root as `task.js` with module.exports (rather than transpiling and exporting a default function)